features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -166,17 +166,33 @@ export default function Resources(props) {
                 }
                 const regEx = /(?<=\{)(?!\s*\{)[^{}]+/g;
```

Comment:
@ruks , I think this regex contains a [lookbehind](https://v8.dev/blog/regexp-lookbehind-assertions) pattern which is [not supported](https://v8.dev/blog/regexp-lookbehind-assertions) in FF, Shall we use an alternative pattern ?

Reply:
yes. seems it is not supported in FF. Will use an alternative patterns.

Reply:
Fixed